### PR TITLE
[Python]Use bullseye instead for buster base images

### LINKED ISSUE #481: [V4 Images] V4 Python images are using python:3.9-slim-buster base images
There is no python:3.9-slim-bullseye image available currently.
Need to confirm if using the buster image is okay from functionality point of view.

+@vrdmr

### PR DESCRIPTION
**Pull request by pragnagopa**
Resolves #481

### CODE REVIEW

File: host/4/bullseye/amd64/python/python37/python37-composite.template [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-buster
+FROM python:3.7-slim-bullseye
```
Comment (pragnagopa) [inline host/4/bullseye/amd64/python/python37/python37-composite.template:1]: @balag0 / @vrdmr  - any reason to use buster instead for bullseye?

Comment (balag0) [reply host/4/bullseye/amd64/python/python37/python37-composite.template:1]: Probably at that time due to - https://github.com/Azure/azure-functions-docker/issues/481

Comment (vrdmr) [reply host/4/bullseye/amd64/python/python37/python37-composite.template:1]: Yeah - thanks @Balag0 for pointing this out. This was due to buster not being available. This should be good to go now.

**Review by vrdmr — APPROVED** [review]
Thanks a lot @pragnagopa for the changes. LGTM

**Comment by pragnagopa** [comment]
Taggging @TsuyoshiUshio as FYI

Tagging @CooperLink - as building App Services is failing.

**Comment by pragnagopa** [comment]
@CooperLink - for PR builds - do we have artifacts that will show generated docker files?

**Comment by CooperLink** [comment]
PR builds failing because bullseye is missing a dependency we had in buster it seems to me.  
'libicu63' is missing.  I will work on resolving this issue tomorrow. I think that I'll just need to add code to install that package.

**Pull request merged**